src/common/modules/PageHandler.js

```diff
@@ -37,7 +37,7 @@ export async function insertIntoPage(text) {
         // send request to insert emoji
         // This will not work in Manifest V3: https://developer.chrome.com/docs/extensions/mv3/intro/mv3-migration/#executing-arbitrary-strings
         return browser.tabs.executeScript(tab.id, {
-            code: `insertIntoPage("${text}");`,
+            code: `insertIntoPage(${JSON.stringify(text)});`,
```

Comment:
Hmm are you sure this works? I know one probably does not want code execution here and this is what it prevents, but is JSON serializing really the correct solution? 

For numbers it may convert them to a string and stuff:
![image](https://github.com/rugk/awesome-emoji-picker/assets/11966684/320d9b50-6f22-4750-9986-d3dcf7375c7d)

Though, that may probably be intended actually…